### ISSUE TITLE
Is this project even still open-source?

### ISSUE THREAD

**Issue by jerkstorecaller** [Tampermonkey/tampermonkey#1545]
Hi. The license assigned to this github repo is GPL3. 

The repo hasn't had any source commits since 2018 (/src directory). The most recent commit is a github issue template in 2021. So essentially, no new code since 2018.

And yet there's still frequent releases, most recently May 2022:  https://addons.mozilla.org/en-US/firefox/addon/tampermonkey/versions

If it's the GPL, the source code should be made available to users, it's how the GPL works. Where is the code for the past 4 years of releases? I looked at tampermonkey.net and it doesn't link to an alternative project hosting website.

If this project has ceased being GPL3 since 2018, please update the license to say "Proprietary" or "Closed Source". And if you want to be ethical put a notice at the beginning of the README that although it used to be GPL, it is now closed-source, so people are not tricked. There's people like me who will never trust a closed-source extension with all their browser history.

**Comment by jerkstorecaller** [comment]
Have you shared the user's browsing history with any 3rd parties? If so, which?

**Comment by jerkstorecaller** [comment]
So the "websites that you visit" refers to the update check page only? You don't have records of other websites that the user might visit, such as playboy.com?

**Comment by derjanb** [comment]
> So the "websites that you visit" refers to the update check page only?

Yes, and the changelog page if opened and when a new version is installed. 

Simply the information that the browser sends in order to fulfill the request, which includes what site should be loaded from the web server and the is logged at the server log files.

**Comment by jerkstorecaller** [comment]
In that case I would rephrase "websites that you visit", it sounds way darker than this  :)

Maybe say "The Tampermonkey webpage accessed when checking for updates or opening changelog".

**Comment by radian628** [comment]
> In that case I would rephrase "websites that you visit", it sounds way darker than this :)
> 
> Maybe say "The Tampermonkey webpage accessed when checking for updates or opening changelog".

I'd like to echo this concern too. Stumbled across this exact same rabbit hole of "what do they mean by 'websites that you visit'?" until I found this thread.
